integration_tests/src/main/python/generate_expr_test.py

```diff
@@ -63,3 +68,33 @@ def test_explode_litarray(data_gen):
             lambda spark : four_op_df(spark, data_gen).select(f.col('a'), f.col('b'), f.col('c'), 
                 f.explode(array_lit)))
 
+#sort locally because of https://github.com/NVIDIA/spark-rapids/issues/84
```

Comment:
This only applies to sorting floating point values.  There are no floating point values in this test so we should remove the comment and sort on the GPU.

Reply:
Done.